hpx-sys/include/wrapper.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <hpx/hpx_init.hpp>
+#include <iostream>
+#include <cstdint>
+#include <vector>
+
+#include "rust/cxx.h"
+
+
+/*inline std::int32_t start() { return hpx::start(nullptr, 0, nullptr); }*/
+
+/*inline std::int32_t start(rust::Fn<int(int, char **)> rust_fn, int argc, char **argv) {*/
+/*	return hpx::start(*/
+/*		[&](int argc, char **argv) {*/
+/*		return rust_fn(argc, argv);*/
+/*        },*/
+/*        argc, argv);*/
+/*}*/
```

Comment:
Could you please remove this? Or is there something you want to make a note of? If that is the case could you please elaborate?